recipe/build-r-base.sh

```diff
@@ -455,14 +345,12 @@ Mingw_w64_makefiles() {
     # echo "Running make check-all (up to 3 times, there is some flakiness in p-r-random-tests.R), this will take some time ..."
     # make check-all -j1 > make-check.log 2>&1 || make check-all -j1 > make-check.2.log 2>&1 || make check-all -j1 > make-check.3.log 2>&1
     cd installer
-    make imagedir
+    make R_PKGS_RECOMMENDED= imagedir
```

Comment:
We only make `imagedir` here and nothing else.
We also don't run `make distribution` above, so I think that we never hit `rinstaller` from `src/gnuwin32/Makefile` and thus not `all`->`$(RPREFIX)-win.exe` in `src/gnuwin32/installer/Makefile`.
Meaning, we might be able to remove all those Inno Setup-related things (`ISDIR`, `isdir`, `DLCACHE`, `win-extra-files`.
I'm not 100% certain, though -- we'd have to try it out; could be done here or in a subsequent PR.

Reply:
Let's try in a subsequent PR. Makes it easier for me to try downstream pacakges.

Reply:
FYI, removing it built fine for https://github.com/conda-forge/r-base-feedstock/pull/309/commits/f9c0ebde157fff6fe7ac7518ac8a1aa8679f890d